docs/actions/actions.md

```diff
@@ -87,15 +87,21 @@ We have also added a Docker Action, that can help you run docker images as an ac
 
 ![img_5.png](img_5.png)
 
-| Option               | Usage                                                    |
-|----------------------|----------------------------------------------------------|
-| docker-image-name    | Required. Image name of the docker image.                |
-| docker-cmd           | Required. Command to run inside the docker image.        |
-| docker-env           | Optional. Environment variables to set in the container. |
-| docker-volume-mounts | Optional. Volume mounts present inside the container.    |
+| Option               | Usage                                                                                                                                                                    |
+|----------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| docker-image-name    | Required. Image name of the docker image.                                                                                                                                |
+| docker-cmd           | Required. Command to run inside the docker image.                                                                                                                        |
+| docker-env           | Optional. Environment variables to set in the container.                                                                                                                 |
+| docker-network       | Optional. Network name for the container. {e.g. "host"}                                                                                                                  |
```

Comment:
I would use description "Connect a container to a network" (similar to `docker run --help`) Otherwise it's easy to confuse with `--hostname` option

Reply:
yeah agreed - I changed it